tutorials/physics/collision_shapes_3d.rst

```diff
@@ -0,0 +1,133 @@
+  shape. Since it genereates multiple shapes, it is more accurate for concave
+  objects at the cost of performance. For objects with medium complexity, it
+  will likely be faster than using a single concave collision shape.
+
+Concave (trimesh) collision shapes
+----------------------------------
+
+:ref:`Concave collision shapes <class_ConcavePolygonShape>` (also called trimesh
+collision shapes) can take any form, from a few triangles to thousands of
+triangles. Concave shapes are the slowest option, but are also the most accurate
+collision shapes available in Godot. **Concave shapes can only be used within
+StaticBodies.** They will not work with KinematicBodies or RigidBodies unless
+the RigidBody's mode is set to Static.
+
+When not using GridMaps for level design, concave collision shapes are usually
+the best approach for level collision. That said, if your level has small
+details, you will probably want to exclude those from collision for better
+performance and reliablity. To do so, you can build a simplified collision mesh
+in a 3D modeller and have Godot generate a collision shape for it automatically
+(see below).
+
+Note that unlike primitive and convex shapes, a concave collision shape doesn't
+have an actual "volume". Objects can be placed both *outside* of the shape as
+well as *inside*.
+
+You can generate a concave collision shape from the editor by selecting a
+MeshInstance and using the **Mesh** menu at the top of the 3D viewport.
+The editor exposes two generation options:
+
+- **Create Trimesh Static Body** is a convenience option. It will create a
+  StaticBody containing a concave shape matching the mesh's geometry.
+
+- **Create Trimesh Collision Sibling** will create a CollisionShape node
+  containing the a concave shape matching the mesh's geometry.
+
+.. note::
+
+    If you need to make a RigidBody *slide* on a concave collision shape, you
+    may notice that sometimes, the RigidBody will bump upwards. To solve this,
+    open the **Project Settings** then create a new boolean property in the
+    ``physics/3d`` category with the name ``smooth_trimesh_collision``. Once
+    you've created this property, enable it by ticking its checkbox.
+
+    Now, make sure the concave shape is the only shape of your StaticBody and
+    that it's at located at its origin without any rotation. This way, the
+    RigidBody should slide perfectly on the StaticBody.
+
+.. seealso::
+
+    Godot can generate collision shapes for your imported 3D scenes
+    automatically. See :ref:`doc_importing_scenes_import_hints` in the
+    documentation for more information.
+
+Performance caveats
+-------------------
+
+While you aren't limited to a single collision shape per PhysicsBody, it's
+recommended to keep the number of shapes as low as possible to improve
+performance. This is especially true for dynamic objects such as RigidBodies and
+KinematicBodies.
+
```

Comment:
Is better so say something like:

While you can create an entire level out of a single `StaticBody`, by placing many shapes, is not advised. Is much better have 1 `CollisionShape` for 1 `StaticBody`, with the shape at the origin (not rotated and not translated).

In this way: the physics engine broad phase algorithm, which is really fast, can discard the body that are not active - the narrow phase will take into account only few of those shapes.
If we would create a `StaticBody` with many shapes, the broad phase would fail and the narrow phase (which is the algorithm that retrieve the various contacts, so it's slow) is forced to perform the collision check for all the shapes.